core/src/avm2/globals/flash/text/StaticText.as

```diff
@@ -1,11 +1,8 @@
 package flash.text {
     import flash.display.DisplayObject;
     
-    [Ruffle(NativeInstanceInit)]
+    [Ruffle(InstanceAllocator)]
     public class StaticText extends DisplayObject {
```

Comment:
Note: `StaticText` should be a final class, should this be changed here?

Reply:
Done